### ISSUE TITLE
[BUG?] Spell points regen on short rest

### ISSUE THREAD

**Issue by NerdyViking** [misthero/dnd5e-spellpoints#88]
I'm assuming it's not supposed to do this, so I figured I'd let you know. The default "Spell Points" item is set to "recharge" on a short rest, rather than a long rest. So, whenever my players short rest, it's giving them all their spell points back! It's an easy fix in game by just unlocking the compendium and editing the item.

**Comment by misthero** [comment]
> I'm assuming it's not supposed to do this, so I figured I'd let you know. The default "Spell Points" item is set to "recharge" on a short rest, rather than a long rest. So, whenever my players short rest, it's giving them all their spell points back! It's an easy fix in game by just unlocking the compendium and editing the item.

yes, that can be done on the character sheet too, unlocking the edit mode and clicking the cog near the spell points bar or, if you want to have a default-ish without editing the compendium, import the item from the compendium, edit that and let the playera drag that on their sheets, you can also have multiple copies of the item with different names and configurations if you need different spellpoint progression or setting for different classes.
